### DISCUSSION TITLE
Is there a way to ignore specific species?

### DISCUSSION THREAD

**Discussion by DaveBKC** [mcguirepr89/BirdNET-Pi#616] [General]
I have a main road about 1/2 mile (800 meters) from the house. When there are sirens on that road, BirdNET-Pi mistakes them for an Eastern Screech Owl.  Rather than pollute the database, is it possible to ignore certain species that we know are always invalid?

**Comment by ehpersonal38** [comment]
You can prevent BirdNET-Pi from reporting predictions for those species by adding them to your "Excluded Species List".

To do this, go to "Tools" > "Excluded Species List"

On that page, press "Ctrl + F" to enter your browser's search mode. Type the name of the species you wish to exclude from your reporting and data collection, select from within the list of species in the box on the left and click ">>ADD>>" to have the species added to your Excluded Species List.

Best of luck.

**Reply by DaveBKC** [reply]
Hot dog! Thanks!
